src/main/java/org/websoso/WSSServer/service/UserService.java

```diff
@@ -97,6 +99,21 @@ public MyProfileResponse getMyProfileInfo(User user) {
         return MyProfileResponse.of(user, avatar, genrePreferences);
     }
 
+    @Transactional(readOnly = true)
+    public ProfileGetResponse getProfileInfo(User visitor, Long ownerId) {
+        User owner = getUserOrException(ownerId);
+        Byte avatarId = owner.getAvatarId();
+        Avatar avatar = avatarRepository.findById(avatarId)
+                .orElseThrow(
+                        () -> new CustomAvatarException(AVATAR_NOT_FOUND, "avatar with the given id was not found"));
+        List<GenrePreference> genrePreferences = genrePreferenceRepository.findByUser(owner);
+
+        if (visitor != null && visitor.getUserId().equals(ownerId)) {
+            return ProfileGetResponse.of(owner, avatar, genrePreferences, true);
+        }
+        return ProfileGetResponse.of(owner, avatar, genrePreferences, false);
```

Comment:
p3; 아래와 같이 변경하면 더 쉽게 파악할 수 있을 것 같습니다!
```suggestion
        boolean isOwner = visitor != null && visitor.getUserId().equals(ownerId);
        return ProfileGetResponse.of(owner, avatar, genrePreferences, isOwner);
```

Reply:
오호, true/false 직접 넣는 것보다 훨씬 좋겠네요!! 굿 👍